src/WingetCreateCLI/Commands/NewCommand.cs

```diff
@@ -307,7 +307,7 @@ private async Task<bool> PromptPackageIdentifierAndCheckDuplicates(Manifests man
             VersionManifest versionManifest = manifests.VersionManifest;
             versionManifest.PackageIdentifier = PromptProperty(versionManifest, versionManifest.PackageIdentifier, nameof(versionManifest.PackageIdentifier));
 
-            GitHub client = new GitHub(null, this.WingetRepoOwner, this.WingetRepo);
+            GitHub client = new GitHub(this.GitHubToken, this.WingetRepoOwner, this.WingetRepo);

```

Comment:
If the user passes in a bad token, this will fail before the user can create their manifest, is that what we want? I think for the new command this shouldn't be blocking, as we should just error out during the submission flow.

Reply:
Resolved offline, if user enters a bad token, we should notify them immediately with a friendly message